api/server/router/container/container.go

```diff
@@ -60,6 +60,7 @@ func (r *containerRouter) initRoutes() {
 		// PUT
 		local.NewPutRoute("/containers/{name:.*}/archive", r.putContainersArchive),
 		// DELETE
+		local.NewDeleteRoute("/containers/_all", r.deleteAllContainers),
```

Comment:
Forgot to say that I would also be okay with "it's client's responsibility to not pass an empty container name" and thus not making a difference between `/containers` and `/containers/`. Otherwise I stand by my proposal so far :)